src/python/devcontainer-feature.json

```diff
@@ -65,15 +65,7 @@
         "ms-python.vscode-pylance"
       ],
       "settings": {
-        "python.defaultInterpreterPath": "/usr/local/python/current/bin/python",
-        "python.formatting.autopep8Path": "/usr/local/py-utils/bin/autopep8",
-        "python.formatting.blackPath": "/usr/local/py-utils/bin/black",
-        "python.linting.flake8Path": "/usr/local/py-utils/bin/flake8",
-        "python.linting.flake8Enabled": false,
-        "python.linting.mypyPath": "/usr/local/py-utils/bin/mypy",
-        "python.linting.mypyEnabled": false,
-        "python.linting.pylintPath": "/usr/local/py-utils/bin/pylint",
-        "python.linting.pylintEnabled": false
+        "python.defaultInterpreterPath": "/usr/local/python/current/bin/python"
```

Comment:
Can we bump the minor version? Thanks! 

https://github.com/devcontainers/features/blob/57c7bec14dca54fde4b4df2580ed429484180882/src/python/devcontainer-feature.json#L3